src/main/java/frc/robot/commands/sequences/AutonAutonSequence.java

```diff
@@ -0,0 +1,312 @@
+            Rotation2d.fromDegrees(0)
+        )
+        ),
+
+        TOPINIT(new Pose2d(
+            Units.inchesToMeters(70.007),
+            Units.inchesToMeters(195.55),
+            Rotation2d.fromDegrees(180)
+        )),
+        BALANCEINIT(
+            new Pose2d(
+            Units.inchesToMeters(70.007),
+            Units.inchesToMeters(107.638),
+            Rotation2d.fromDegrees(180)
+        )
+        ),
+        BOTTOMINIT(
+            new Pose2d(
+            Units.inchesToMeters(70.007),
+            Units.inchesToMeters(12.873),
+            Rotation2d.fromDegrees(180)
+        )
+        );
+
+        Pose2d blue;
+
+        Blue(Pose2d blue){
+            this.blue = blue;
+        }
+    } 
+
+
+    //non balancing auton sequence
+    public AutonAutonSequence(RobotContainer robotContainer, Pose2d initialPose, Pose2d placePose, Pose2d grabPose, Pose2d placePose2, MoverPosition moverPosition, MoverPosition moverPosition2){
+        this.robotContainer = robotContainer;
+
+        swerveSubsystem = robotContainer.getSwerveSubsystem();
+        gripperSubsytem = robotContainer.getGripperSubsytem();
+        moverSubsystem = robotContainer.getMoverSubsystem();
+ 
+        addRequirements(swerveSubsystem, gripperSubsytem, moverSubsystem);
+
+        addCommands(
+            //place preloaded gamepiece
+            goAndPlace(initialPose, placePose, moverPosition),
+            //go and grab 2nd piece
+            goAndGrab(placePose, grabPose),
+            //go and place grabbed piece
+            goAndPlace(grabPose, placePose2, moverPosition2)
+
+        );
+    }
+    // balancing auton sequence
+    public AutonAutonSequence(RobotContainer robotContainer, Pose2d initialPose, Pose2d placePose, Pose2d outsidePose, MoverPosition moverPosition){
+        this.robotContainer = robotContainer;
+
+        swerveSubsystem = robotContainer.getSwerveSubsystem();
+        gripperSubsytem = robotContainer.getGripperSubsytem();
+        moverSubsystem = robotContainer.getMoverSubsystem();
+
+        addRequirements(swerveSubsystem, gripperSubsytem, moverSubsystem);
+
+        addCommands(
+            //place pre loaded gamepiece
+            goAndPlace(initialPose, placePose, moverPosition),
+            //go out of community
+            new FollowPathCommand(swerveSubsystem, placePose, null, outsidePose),
+            //go and balance on charging station
+            new BalancerCommand(swerveSubsystem)
+        );
+
+    }
+
+    //make followpath and Shiralevel parallel commands?
+
+    public Command goAndGrab(Pose2d initialPose, Pose2d finalPose){
+        addCommands(        
+            //get to gamepiece
+            new FollowPathCommand(swerveSubsystem, initialPose, null, finalPose),
+            //get mover to ground height
+            new ShiraLevelCommand(moverSubsystem, MoverPosition.GROUND),
+            //grab game piece
+            new MatthewIntakeCommand(gripperSubsytem)
+        );
+        return null;
```

Comment:
If you're returning `null`, just make this method void (and don't call `addCommands()` on the result of this method). Alternatively, make this return the sequential command sequence from chaining these via `.andThen()`.